sdk/storage/azure-storage-file-datalake/tests/testcase.py

```diff
@@ -172,73 +146,22 @@ def _set_test_proxy(service, settings):
                 settings.PROXY_PASSWORD,
             )
 
-    def _get_shared_key_credential(self):
+    def _get_shared_key_credential(self, account_name, account_key):
```

Comment:
it's a type of credential we can pass in DataLakeServiceClient etc. 
It seems currently we are not using this credential, an we passed account key as credential.
While it would be better if we can use _shared_key_credential in dict format especially for customized url. Since with a pure account key we cannot parse out the account name for user.